infrastructure/code/code_html.go

```diff
@@ -43,11 +45,50 @@ func getDataFlowHtml(issue snyk.CodeIssueData) string {
 		  <span class="data-flow-filepath">%s:%d</span>
 		  <span class="data-flow-delimiter">|</span>
 		  <span class="data-flow-text">%s</span>
-		</div>`, i+1, flow.FilePath, flow.FlowRange.Start.Line+1, flow.Content)
+		</div>`, i+1, getFileName(flow.FilePath), flow.FlowRange.Start.Line+1, flow.Content)
 	}
 	return dataFlowHtml
 }
 
+func getCodeDiffHtml(fix snyk.ExampleCommitFix) string {
+	linesHtml := ""
+	for _, commit := range fix.Lines {
+		linesHtml += fmt.Sprintf(`
+		<div class="example-line %s"><code>%s</code></div>`, commit.LineChange, commit.Line)
+	}
+	return linesHtml
+}
```

Comment:
Example of how to use html/template. Though instead of doing this, you'll likely want to declare multiple named templates (using `{{ define ...`) and then use [ExecuteTemplate](https://pkg.go.dev/html/template#Template.ExecuteTemplate) to invoke them by name. This would be preferable to inserting rendered template strings into other templates (since they will be default HTML-escaped by design). 

```suggestion
var codeDiffTmpl = template.Must(template.New("codediff").Parse(`
{{- range .Lines }}
  <div class="example-line {{.LineChange}}><code>{{.Line}}</code></div>
{{- end }}
`)

func getCodeDiffHtml(fix snyk.ExampleCommitFix) (string, error) {
  var out bytes.Buffer
  err := codeDiffTmpl.Execute(&out, fix)
  if err != nil {
    return "", err
  }
  return out.String(), nil
}
```